go/worker/storage/committee/checkpoint_sync.go

```diff
@@ -174,16 +174,17 @@ func (n *Node) nodeWorker(
 				"root", chunk.Root,
 				"err", result.err,
 			)
-			fallthrough
-		case errors.Is(result.err, checkpoint.ErrChunkCorrupted):
-			chunkReturnCh <- chunk
-			return result.err
-		case errors.Is(result.err, checkpoint.ErrChunkProofVerificationFailed):
-			errorCh <- checkpointStatusNext
-			return backoff.Permanent(result.err)
-		case result.err != nil:
-			errorCh <- checkpointStatusBail
-			return backoff.Permanent(result.err)
+			switch {
```

Comment:
Should this be backported to `21.1.x`? It looks like a bugfix as the previous code would not skip a bad checkpoint?

Reply:
The effect here is just to shorten the failure path. Without this fix, it would keep retrying up to `maxRetries` and then fail properly; with this, it stops immediately, as the original intention was.